frontend/src/scenes/insights/views/LineGraph/LineGraph.tsx

```diff
@@ -689,14 +686,6 @@ export function LineGraph_({
                             const height = dynamicHeight
                             const parentNode: any = scale.chart?.canvas?.parentNode
                             parentNode.style.height = `${height}px`
-                        } else if (shouldAutoResize) {
```

Comment:
This was only used by the bar value graph and made no sense, as it is generally prferable that it fits the bounds which is what the default does